docs/api-reference/next/image.md

```diff
@@ -14,182 +14,222 @@ description: Enable Image Optimization with the built-in Image component.
+| [`onLoadingComplete`](#onLoadingComplete) | `onLoadingComplete={img => done())}` | Function        | -        |
+| [`onLoad`](#onLoad)                       | `onLoad={event => done())}`          | Function        | -        |
+| [`onError`](#onError)                     | `onError(event => fail()}`           | Function        | -        |
+| [`loading`](#loading)                     | `loading="lazy"`                     | String          | -        |
+| [`blurDataURL`](#blurDataURL)             | `blurDataURL="data:image/jpeg..."`   | String          | -        |
+
+### Required Props
+
+The Image Component requires the following properties: `src`, `width`, `height`, and `alt`.
+
+```jsx:app/page.js
+import Image from 'next/image';
+
+export default function Page() {
+  return (
+    <div>
+      <Image
+        src="/profile.png"
+        width={500}
+        height={500}
+        alt="Picture of the author"
+      />
+    </div>
+  );
+}
+```
 
-### src
+#### `src`
 
 Must be one of the following:
 
-1. A [statically imported](/docs/basic-features/image-optimization.md#local-images) image file, or
-2. A path string. This can be either an absolute external URL,
-   or an internal path depending on the [loader](#loader) prop.
+1. A [locally imported](/docs/basic-features/image-optimization.md#local-images) image file. Or,
+2. A path string. This can be either an external URL or an internal path.
 
-When using an external URL, you must add it to [remotePatterns](#remote-patterns) in `next.config.js`.
+> **Good to know**: When using an external URL, you must add it to [remotePatterns](##remotepatterns) option in `next.config.js`.
 
-### width
+#### `width`
 
-The `width` property represents the _rendered_ width in pixels, so it will affect how large the image appears.
+The `width` property represents the rendered width in pixels.
 
-Required, except for [statically imported images](/docs/basic-features/image-optimization.md#local-images) or images with the [`fill` property](#fill).
+This property is required **except** for [local images](/docs/basic-features/image-optimization.md#local-images) or images with the [`fill` property](#fill).
 
-### height
+#### `height`
 
-The `height` property represents the _rendered_ height in pixels, so it will affect how large the image appears.
+The `height` property represents the rendered height in pixels.
 
-Required, except for [statically imported images](/docs/basic-features/image-optimization.md#local-images) or images with the [`fill` property](#fill).
+This property is required **except** for [local images](/docs/optimizing/image.md#local-images) or images with the [`fill` property](#fill).
 
-### alt
+#### `alt`
 
 The `alt` property is used to describe the image for screen readers and search engines. It is also the fallback text if images have been disabled or an error occurs while loading the image.
```

Comment:
I noticed you added examples to all the others except `alt`

Reply:
This is because of the following paragraph:

> If the image is [purely decorative](https://html.spec.whatwg.org/multipage/images.html#a-purely-decorative-image-that-doesn't-add-any-information) or [not intended for the user](https://html.spec.whatwg.org/multipage/images.html#an-image-not-intended-for-the-user), the `alt` property should be an empty string (`alt=""`).